charts/camunda-platform/templates/console/_helpers.tpl

```diff
@@ -63,3 +63,18 @@ Get the image pull secrets.
         "context" $
     ) -}}
 {{- end }}
+
+{{/*
+[console] Define variables related to authentication.
```

Comment:
Any reason to wrap the plain vars in the named templates?
If not, let's use them directly in the configmap.yaml

Reply:
@aabouzaid  Not sure what you mean ;) I just need the vars in the configmap.

If you could make the adjustments I'd be more than happy.

Reply:
@urbanisierung my question about doing this:

```
clientId: {{ include "console.authClientId" . | quote }}
```

not this:
```
clientId: {{- .Values.global.identity.auth.console.clientId | quote -}}
```

the point here, it's working like a var, but used in 1 place only.
do you have any plans to use it in different places or extend the logic?

Reply:
@aabouzaid  Ah got it. So this is aligned with the implementation of the other components, like for example with web modeler [here](https://github.com/camunda/camunda-platform-helm/blob/dd2b3fe3391cc7b9e568a8dbd1dc820ac903fc84/charts/camunda-platform/templates/web-modeler/_helpers.tpl#L329).

And with that aligned with the config in the [docs](https://docs.camunda.io/docs/next/self-managed/setup/guides/connect-to-an-oidc-provider/?optionsType=helm).

Let me know if I'm wrong with that.

> do you have any plans to use it in different places or extend the logic?

But to answer your question: no, will not use the variable out of console. But I doubt the corresponding auth variables from the other components are used somewhere else.

Reply:
Maybe WM had plans to extend that.
Given no plans to extend that, let's use the vars directly like `clientId: {{- .Values.global.identity.auth.console.clientId | quote -}}`

Reply:
@aabouzaid So that's how it's set right now. Can we resolve this then?

Reply:
I've added it here: https://github.com/camunda/camunda-platform-helm/pull/1782/commits/3ab6f1b08d4f4fb317475a949693c706151139f8